src/main/java/gregtech/api/capability/impl/miner/MinerLogic.java

```diff
@@ -164,22 +164,24 @@ public void performMining() {
             NonNullList<ItemStack> blockDrops = NonNullList.create();
             IBlockState blockState = metaTileEntity.getWorld().getBlockState(blocksToMine.getFirst());
 
-            // if the block is not air or cobblestone., harvest it
-            if (GTUtility.isOre(GTUtility.toItem(blockState))) {
+            // check to make sure the ore is still there,
+            while(!GTUtility.isOre(GTUtility.toItem(blockState))) {
```

Comment:
If you go though list here and find nothing to mine this will still drain energy for mining operation as that was done earlier. If we want to be completely correct about this, parts of this function will need to be shuffled around. Though this shouldn't happen that often to be a problem. So I won't block this PR.